demo/apisix/conf/apisix.yaml

```diff
@@ -8,16 +8,22 @@ routes:
     plugin_config_id: 1
   -
     id: jmap
-    uri: /jmap
+    uri: /oidc/jmap
```

Comment:
Should we add `/oidc/` prefix for all uri?
In this commit, It was added for `/oidc/jmap` and `/oidc/jmap/session`

// I tried to read krakend.json, but don't see the related

Reply:
> Should we add /oidc/ prefix for all uri?

I think yes.
rf: https://gitlab.com/linagora_private/saas/tools/helm-charts/helm-krakend/-/blob/main/krakend/configs/settings/oidc_endpoints.json